src/spikeinterface/core/generate.py

```diff
@@ -793,20 +795,20 @@ def synthesize_random_firings(
 
     Parameters
     ----------
-    num_units : int
+    num_units : int, default: 20
         Number of units.
-    sampling_frequency : float
+    sampling_frequency : float, default: 30000.0 (in Hz)
```

Comment:
For these sampling frequency default values I don't know if they should be (in Hz) vs having the (in Hz) down in the actual description. So for me I would prefer:

```
sampling_frequency : float, default: 30000.0
    The sampling rate entered in Hz. (or samples/second)
```

opinions @chrishalcrow @h-mayorquin ?

Reply:
I'd personally prefer the Hz to be in the description, so that a user can copy and paste the default value and it would work.

Reply:
Agree I prefer in the description I will make the change